langserver/session/types.go

```diff
@@ -5,12 +5,15 @@ import (
 	"log"
 
 	"github.com/creachadair/jrpc2"
+
+	"github.com/hashicorp/terraform-ls/internal/terraform/discovery"
 )
 
 type Session interface {
 	Assigner() (jrpc2.Assigner, error)
 	Finish(jrpc2.ServerStatus)
 	SetLogger(*log.Logger)
+	SetDiscovery(discovery.DiscoveryFunc)
```

Comment:
Nitpick: Can we call this `SetDiscoveryFunc`, so it's more obvious that the argument is a function?